jablib/src/main/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheck.java

```diff
@@ -55,6 +53,19 @@ public class BibliographyConsistencyCheck {
             StandardField.MODIFICATIONDATE
     );
 
+    private final BibEntryTypesManager bibEntryTypesManager;
+
+    // Default constructor
```

Comment:
Comment does not add value - for what is the constructor used?

I think, this constructor should be removed - only the other consturctor should be used by the normal JabRef code - and passing down the bibentrytypes manger? (It is available through `org.jabref.logic.preferences.JabRefCliPreferences#getCustomEntryTypesRepository`)